### PR TITLE
Checking & Exploring

### PR DESCRIPTION
**Pull request by hxmattobee**
My first post for the blog. Exploratory software testing has matured and grown to encompass a wide range of activities. Meanwhile, computers can be automated to perform highly efficient and reliable checks without human intervention. How can we combine these two approaches to achieve the best results?

### CODE REVIEW

**Comment by drumheadrob** [comment]
Really good article. Thumbs up from me!

**Comment by hpoom** [comment]
:+1: Enjoyed reading it. Probably worth getting a grammar bod to read over it.

File: _posts/2015-08-19-checking-and-exploring.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+> Checking is the process of making evaluations by applying algorithmic decision rules to specific observations of a product.
+
+Algorithmic decision rules? It's true that checking is boring and so it often makes sense to have computers do it instead of people. The temptation therefore is to automate everything and be done with it. If a little is good, more must be better, right? The thing is, checks on their own—whether performed by a computer or a person—can't reveal all of the bugs, let alone all of the risks and opportunities that exist in complex software.
+
+Checking uses validation and verification to answer the question, "does this application do everything that it’s supposed to do?". To be reliable, checks have to be logical and repeatable, but that level of formality doesn't leave any room for curiosity, experminentation or empathy. Neither does it allow for the random variations in human behaviour (e.g. clicking buttons at slighly different speeds, providing slightly different inputs or taking slightly different routes) that so often lead to the discovery of bugs.
```
Comment (Joezo) [inline _posts/2015-08-19-checking-and-exploring.md]: should `experminentation` be `experimentation`?

**Comment by Joezo** [comment]
Really good post Matt :)

**Comment by jackdcrawford** [comment]
This is great :+1: really enjoyed reading

File: _posts/2015-08-19-checking-and-exploring.md (hunk 1 above)
Comment (hxmattobee) [inline _posts/2015-08-19-checking-and-exploring.md]: Experminentation! Good spot.

**Pull request merged**